examples/checkout-book.yaml

```diff
@@ -0,0 +1,93 @@
+'@context':
+  - https://git.io/vf-examples-jsonld-context
+  - library: https://library.example/
+    bob: https://bob.example/
+
+
+'@id': rgh:valueflows/valueflows/master/examples/checkout-book.yaml
+'@graph':
+
+  # resource before
+  - '@id': library:2402b57c-1c45-423a-acb2-f50961c97d65
+    '@type': EconomicResource
+    skos:note: Valueflows cookbook
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 2
+
+  - '@id': library:21f361a6-2375-46bb-b192-c21b5ba833bf
+    '@type': EconomicResource
+    skos:note: Possession rights and responsibilities for the Valueflows cookbook
```

Comment:
How one can know what rights it represent? Would you have different resource to represent each different rights?

Reply:
>Would you have different resource to represent each different rights?

I think so.  At least everything that this agent would need for that underlying resource.  Could be combinations of rights.  Haven't thought more about how to define R&R, but leaning towards something like POA but improved, along with reference to agreements.